main.py

```diff
@@ -0,0 +1,15 @@
+# this allows us to use code from
+# the open-source pygame library
+# throughout this file
+import pygame
+
+
+def main():
+    try:
+        pygame.init()
+        print('Starting asteroids!')
+    except Exception as e:
+        print(f'Failed to initialize Pygame: {e}')
+
+if __name__ == "__main__":
+    main()
```

Comment:
**Suggestion:** Add a return statement in the `main` function for clarity.

**Existing Code:**
```Python
main()
```

**Improved Code:**
```Python
return 0
```

**Details:**
Consider adding a return statement in the `main` function to indicate successful completion.